base/regex.jl

```diff
@@ -236,6 +248,19 @@ function show(io::IO, s::SubstitutionString)
     show(io, s.string)
 end
 
+"""
+    @s_str -> SubstitutionString
+
+Construct a substitution string, used for regular expression substitutions.  Within the
```

Comment:
Need to escape all the backslashes?

Reply:
Actually you need single backslashes; the examples don't work with double ones. That's because non-standard string literal macros are passed the raw string, and escapes are not interpreted.

Reply:
The example is inside a julia string.

Reply:
My bad, I chose the wrong thread. My comment applies to uses of `s"..."` below.

Reply:
Where? Everything here is inside a normal julia string.

Reply:
This example no longer works due to double slashes: https://github.com/JuliaLang/julia/pull/27134/files#diff-7422507bb03a8ecee35360dc9380b9caR261

Reply:
But it won't have double slashes in the rendered documentation?